abi-bindings/README.md

```diff
@@ -1,3 +1,7 @@
 ## ABI Bindings
 
-This directory contains ABI bindings for the Solidity contracts in the `contracts/src/CrossChainApplications` and `contracts/src/Teleporter` directories. The files with the same name as the Solidity source files are automatically generated by the `scripts/abi_bindings.sh` script. Other files in this directory (such as the packing utilities) are manually created and maintained.
\ No newline at end of file
+This directory contains ABI bindings for the Solidity contracts in the `contracts/src/CrossChainApplications` and `contracts/src/Teleporter` directories. The files with the same name as the Solidity source files are automatically generated by the `scripts/abi_bindings.sh` script. Other files in this directory (such as the packing utilities) are manually created and maintained.
+
+## Packing
+
+`packing.go` files in individual subfolders define utilities for ABI packing instances of structs auto-generated by `abigen` as well as method calls. For structs we implement `ABIPacker` interface defined in `./packer/packer.go` and add references to the `packer_test.go` file to ensure that our tests are exhaustive and don't fail silently if additional fields are added to the structs in the future on the Solidty side.
```

Comment:
generally have tried to avoid "we" and "our"

Reply:
Thanks. Made the update.